AppCenter/AppCenter/Internals/Util/MSUtility+StringFormatting.m

```diff
@@ -12,8 +12,20 @@
  */
 NSString *MSUtilityStringFormattingCategory;
 
+// We support the following formats:
+// target=<..>
+// appsecret=<..>
+// target=<..>;appsecret=<..>
+// ios=<..>;macos=<..>
+// targetIos=<..>;targetMacos=<..>
```

Comment:
Let's not worry about the default target at this point. It will simplify the logic. I updated the WI.